teamcity/assets/service_checks.json

```diff
@@ -1 +1,75 @@
-[]
+[
+    {
+        "agent_version": "7.40.0",
+        "integration": "Teamcity",
+        "check": "teamcity.openmetrics.health",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "host",
+            "endpoint"
+        ],
+        "name": "OpenMetrics endpoint health",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the OpenMetrics endpoint, otherwise returns `OK`."
+    },
+    {
+        "agent_version": "7.41.0",
+        "integration": "Teamcity",
+        "check": "teamcity.build.status",
```

Comment:
Can you indicate in the description that these are emitted by the rest instance?